.pylint-ignore.dict

```diff
@@ -0,0 +1,21 @@
+param
+sa
+numpy
+py
+args
+kwargs
+ui
+bool
+reimplemented
+gil
+osx
+traceback
+shaders
+whitespace
+gettext
+i18n
+catalog
```

Comment:
I find it weird that it doesn't know `catalog`. What dialect of English was it using, I wonder?